.github/workflows/check-catalyst.yaml

```diff
@@ -443,7 +501,6 @@ jobs:
 
     - name: Run Python Pytest Tests
       run: |
-        # export ASAN_OPTIONS=detect_odr_violation=0
```

Comment:
Do we know why these are no longer an issue or did they just magically disappear? 😅

Reply:
I didn't work on this till yesterday. I thought that you know the answer :smile: will dig in today.

Reply:
Well at the time our tests would through odr violations (one definition rule), but since that didn't seem to have any adverse effects we just disabled the check for the time being. It was unclear what was causing the error.

Since you removed the config that disables the check and reenabled the sanitizer, but the error is not reappearing, it appears that it solved itself? 🤔